engine-precompiles/src/native.rs

```diff
@@ -228,6 +236,16 @@ impl Precompile for ExitToNear {
         context: &Context,
         is_static: bool,
     ) -> EvmPrecompileResult {
+        #[cfg(feature = "error_refund")]
+        fn parse_input(input: &[u8]) -> (Address, &[u8]) {
+            let refund_address = Address::from_slice(&input[1..21]);
+            (refund_address, &input[21..])
```

Comment:
Just FYI, its a good idea that we start setting the end byte len instead of implying them. May result in unwanted behavior.